src/components/subscriptions/SubscriptionDetailPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import SubscriptionExpirationBanner from './expiration/SubscriptionExpirationBanner';
+import SubscriptionExpirationModal from './expiration/SubscriptionExpirationModal';
+import SubscriptionDetails from './SubscriptionDetails';
+import LicenseAllocationDetails from './licenses/LicenseAllocationDetails';
+import { SubscriptionContext } from './SubscriptionData';
+import SubscriptionDetailContextProvider from './SubscriptionDetailContextProvider';
+import { NotFound } from '../NotFoundPage';
+
+const SubscriptionDetailPage = ({ match }) => {
+  const { params: { subscriptionUUID } } = match;
+  const { data: subscriptions } = useContext(SubscriptionContext);
+  if (!subscriptions?.count) {
```

Comment:
I'd move line 19 and 20 to be with the other consts and update this if statement to also check for subscription. 
`if (!subscriptions?.count || !subscription) {`

Currently when an admin who has subscriptions tries to load one that doesn't exist, like a mistyped uuid, the NotFound page doesn't display and the SubscriptionDetailContextProvider gets returned and it causes a whole bunch of errors and the entire page goes blank.

Reply:
good catch!

Reply:
I'm also relocating this logic to be inside `SubscriptionDetailContextProvider`, there was no need to have it split between the parent component and context provider

Reply:
edit: apparently there is a good reason to return the 404 from the parent component - React doesn't like it when a component conditionally calls hooks. `hasMultipleSubscriptions` ended up being the only part of this moved into the context provider

Reply:
Ah, eyes. That is one of the rules of React hooks, that you can't conditionally call hooks as they much execute in the exact same order on every render of the component!